lib/pageloader_builder.dart

```diff
@@ -0,0 +1,20 @@
+// Copyright 2017 Google Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'package:build/build.dart';
+import 'package:source_gen/source_gen.dart';
+
+import 'src/generators/pageobject_generator.dart';
+
+Builder pageloaderBuilder([_]) =>
```

Comment:
Some comment for this API. Do we need the empty params?

Reply:
Yup. Trying to call 'pub run build_runner build' complains when I remove it.

Reply:
Yeah it must be a `BuilderFactory` even if you don't take any options.

https://pub.dartlang.org/documentation/build/0.12.5/build/BuilderFactory.html